lib/rpm_package.py

```diff
@@ -103,7 +103,7 @@ def update_version(self, new_version):
 
         self.write_content()
 
-    def bump_release(self, change_log_lines, user_name, user_email):
+    def update_release_and_change_log(self, change_log_lines, user_name, user_email):
```

Comment:
Historical information: 'bump' is not the same as 'update' because bump is usually used to mean 'increment'. I used 'update' because the version might be totally wrong since we don't have control over what the package owners do.

Regardless, I don't see how bumping the release would not cause the change log to be updated or how updating the change log would not cause the release to be bumped.